<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dongdongju Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
	font: normal 12px Helvetica, Arial, sans-serif;
}
#mainFrame {background: white;}
#upperLabel {
	font: normal 20px Helvetica, Arial, sans-serif;
	color: purple;
}
#lowerLabel {
	color: gray;
}
#loginFrame {
	padding: 9px 10px;
	color: black;
	background: white;
	border: 1px solid #d8d8d8;
	border-radius: 2px;
	padding: 0px 10px;
}
QPushButton {
	padding: 8px 16px;
	font: bold 12px Helvetica, Arial, sans-serif;
	color: #6e6e6e;
	background: #f3f3f3;
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #F5F5F5, stop: 1 #F1F1F1);
	border: 1px solid #d8d8d8;
	border-radius: 2px;
}
QPushButton:hover {
	color: #333;
	border-color: #c6c6c6
}
QPushButton:pressed {
	color: #000;
	border: 1px solid #c6c6c6;
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #eee, stop: 1 #e0e0e0);
}
QTextEdit {
	padding: 7px 10px;
	color: black;
	background: white;
	border: 1px solid #d8d8d8;
	border-radius: 2px;
}
QListWidget {
	padding: 7px 10px;
	color: black;
	background: white;
	border: 1px solid #d8d8d8;
	border-radius: 2px;
}
QLineEdit {
	padding: 7px 10px;
	color: black;
	background: white;
	border: 1px solid #d8d8d8;
	border-radius: 2px;
}</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="chatPage">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QTextEdit" name="teChat">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/thisthis.jpg);
</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="tabChangesFocus">
                <bool>false</bool>
               </property>
               <property name="lineWrapMode">
                <enum>QTextEdit::WidgetWidth</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Helvetica,Arial,sans-serif'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="lwUsers">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/yeah.jpg);</string>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="movement">
                <enum>QListView::Static</enum>
               </property>
               <property name="flow">
                <enum>QListView::TopToBottom</enum>
               </property>
               <property name="layoutMode">
                <enum>QListView::SinglePass</enum>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="viewMode">
                <enum>QListView::ListMode</enum>
               </property>
               <property name="sortingEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="leMessage">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbSend">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>전송</string>
               </property>
               <property name="default">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="loginPage">
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0" alignment="Qt::AlignHCenter">
            <widget class="QFrame" name="loginFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">border-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="IDlabel">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>아이디</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="PWlable">
                    <property name="styleSheet">
                     <string notr="true">border-color: rgb(0, 0, 0);</string>
                    </property>
                    <property name="text">
                     <string>비밀번호</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="leID">
                    <property name="minimumSize">
                     <size>
                      <width>170</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="maxLength">
                     <number>15</number>
                    </property>
                    <property name="clearButtonEnabled">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lePW">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="maxLength">
                     <number>12</number>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="clearButtonEnabled">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QPushButton" name="pbLogin">
                  <property name="styleSheet">
                   <string notr="true">border-color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>로그인</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbSignup">
                  <property name="styleSheet">
                   <string notr="true">border-color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>회원가입</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>705</width>
               <height>438</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="icons.qrc">:/drunkenele.jpg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::NoTextInteraction</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lowerLabel">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>by dongdongju in Dongguk University</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leID</tabstop>
  <tabstop>lePW</tabstop>
  <tabstop>pbLogin</tabstop>
  <tabstop>pbSignup</tabstop>
  <tabstop>teChat</tabstop>
  <tabstop>lwUsers</tabstop>
  <tabstop>leMessage</tabstop>
  <tabstop>pbSend</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
